notebooks/Dynamics.ipynb

```diff
@@ -0,0 +1,295 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dynamics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this notebook we present some of NetworKit dynamic algorithms to analyze properties of a (dynamic) graph.\n",
+    "See the [networkit.dynamics](https://networkit.github.io/dev-docs/python_api/dynamics.html) module for a more detailed description. Dynamic algorithms can compute the result for the adapted graph directly from the previous result, without the need to re-run the entire algorithm.\n",
+    "\n",
+    "Note: The run() method does not need to be called in order to adapt the results!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# As always, the first step is to import NetworKit.\n",
+    "import networkit as nk "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The Graph and Graph Events"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: In order to maintain consistent results, the `Graph` and the `Dynamic Algorithm` needs to be adjusted. \\\n",
```

Comment:
In the introduction `result` (singular) is used. From here on out, you use `results` (plural). Better be consistent throughout one notebook.